pom.xml

```diff
@@ -143,8 +155,6 @@
           </execution>
         </executions>
         <configuration>
-          <!-- get better with time -->
-          <ratchetFrom>origin/master</ratchetFrom>
```

Comment:
Needed to remove this as Jenkins CI does shallow clones: https://ci.jenkins.io/blue/organizations/jenkins/Plugins%2Fdashboard-view-plugin/detail/PR-184/2/pipeline/105#step-137-log-314

```
[2022-07-21T21:36:28.701Z] [ERROR] Failed to execute goal com.diffplug.spotless:spotless-maven-plugin:2.22.8:check (default) on project dashboard-view: Execution default of goal com.diffplug.spotless:spotless-maven-plugin:2.22.8:check failed: No such reference 'origin/master' -> [Help 1]
[2022-07-21T21:36:28.701Z] org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal com.diffplug.spotless:spotless-maven-plugin:2.22.8:check (default) on project dashboard-view: Execution default of goal com.diffplug.spotless:spotless-maven-plugin:2.22.8:check failed: No such reference 'origin/master'
```